micromamba/tests/test_create.py

```diff
@@ -757,6 +758,7 @@ def test_env_dir_idempotence(tmp_path, tmp_home, tmp_root_prefix):
         assert Path(condarc_envs_dirs / env_name).exists()
 
 
+@pytest.mark.parametrize("conda_envs_x", ("CONDA_ENVS_DIRS", "CONDA_ENVS_PATH"))
```

Comment:
We should also test having both of the environment variables set and check the precedence?

Reply:
Sure, I wasn't sure the behavior was defined, but I'll check what mamba 1 and conda do when both are set.

Reply:
Makes sense.  I started looking into it and realized there's another incompatibility in the syntax. 

In conda and mamba1, both `CONDA_ENVS_DIRS` and `CONDA_ENVS_PATHS` are path-separator delimited (colon on Linux, semicolon on Windows).

In mamba2, `CONDA_ENVS_DIRS` is comma-delimited instead.

Reply:
Ok: submitted #3854 to address the case if someone sets both env vars and #3855 to unify the behavior using the path separator as delimiter.